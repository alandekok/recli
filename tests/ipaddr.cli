ip 1.2.3.4
ip 0.1.2.3
ip -3.3
ip xyz
ip 333.0.1.2
ip 1.2.3.3a
