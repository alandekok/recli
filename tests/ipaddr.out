ip -3.3
   ^ Parse error.
ip xyz
   ^ Parse error.
ip 333.0.1.2
   ^ Parse error.
ip 1.2.3.3a
   ^ Parse error.
